Cython/Compiler/TypeStubGenerator.py

```diff
@@ -0,0 +1,433 @@
+from . import Version
+from .Nodes import CNameDeclaratorNode
+from .ExprNodes import CallNode, NameNode, ImportNode, TupleNode, AttributeNode
+from ..CodeWriter import DeclarationWriter
+from .Visitor import CythonTransform
+from . import PyrexTypes
+from ..Utils import open_new_file
+import cython 
+import os 
+import sys 
+
+cython.declare(PyrexTypes=object, Naming=object, ExprNodes=object, Nodes=object,
```

Comment:
Probably not useful unless you intend this file to be compiled with Cython. I'd think it shouldn't be at this stage - it's not a "core" feature so doesn't really need speeding up. If you do intend it to be compiled then the file needs adding to `setup.py`.